.travis.yml

```diff
@@ -3,38 +3,54 @@ node_js:
 install:
-    - yarn install --frozen-lockfile
-    - yarn cy:verify
-    - yarn build
+  - git clone --single-branch --branch gh-pages "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${FEEDBACK_REPO}" feedback-ci
+  - yarn install --frozen-lockfile
+  - yarn cy:verify
+  - yarn test
+  - yarn start &
 script:
-    - PORT=8081 REACT_APP_DHIS2_BASE_URL=http://localhost:8080 REACT_APP_CYPRESS=true yarn start &
-    - yarn wait-on http-get://localhost:8081
-    - yarn wait-on http-get://localhost:8080
-    - CYPRESS_EXTERNAL_API=http://localhost:8080 CYPRESS_ROOT_URL=http://localhost:8081 yarn cy:e2e:run #--record --key $CYPRESS_KEY
-    - kill $(jobs -p) || true
+  - yarn wait-on http-get://localhost:8080
+  - yarn wait-on http-get://localhost:8081
+  - yarn node cypress/support/runner.js -b chrome
+after_script:
+  - cd feedback-ci/
+  - if [[ $TRAVIS_PULL_REQUEST_BRANCH != "" ]]; then BRANCH="${TRAVIS_PULL_REQUEST_BRANCH}"; else BRANCH="${TRAVIS_BRANCH}"; fi;
+  - GITHUB_PAGES_ROOT=$(http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} https://api.github.com/repos/${FEEDBACK_REPO}/pages | jq ".html_url" | tr -d '"')
+  - GITHUB_PAGES_PATH=$(cat ../pages_path)
+  - echo "- Built ``${TRAVIS_REPO_SLUG} ${TRAVIS_COMMIT_MESSAGE}`` against branch ${BRANCH} on $(date). [Report](${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH})" >> README.md
+  - git add . && git commit -m "Results for $TRAVIS_EVENT_TYPE on $TRAVIS_COMMIT"
+  - git fetch origin
+  - git rebase -s ours origin/gh-pages
+  - git push "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${FEEDBACK_REPO}" gh-pages -f
+  - EXISTING_COMMENT=$(http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments | jq '[.[] | select(.user.login == "EyeSeeTeaBotTest")][0].id')
+  - if [[ $TRAVIS_TEST_RESULT == 0 ]]; then MESSAGE_TITLE="Tests ran successfully :tada:"; else MESSAGE_TITLE="Tests failed with errors :rotating_light:"; fi;
+  - if [[ $TRAVIS_PULL_REQUEST != "false" && $EXISTING_COMMENT == "null" ]]; then http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} POST https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments body="<h3>${MESSAGE_TITLE}</h3><br />Build commit ${TRAVIS_COMMIT_MESSAGE} with commit id ${TRAVIS_COMMIT} on branch ${BRANCH}<br /><br />Travis log available at ${TRAVIS_BUILD_WEB_URL}<br /><br />See cypress results at ${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH}"; fi;
+  - if [[ $TRAVIS_PULL_REQUEST != "false" && $EXISTING_COMMENT != "null" ]]; then http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} PATCH https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/comments/${EXISTING_COMMENT} body="<h3>${MESSAGE_TITLE}</h3><br />Build commit ${TRAVIS_COMMIT_MESSAGE} with commit id ${TRAVIS_COMMIT} on branch ${BRANCH}<br /><br />Travis log available at ${TRAVIS_BUILD_WEB_URL}<br /><br />See cypress results at ${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH}"; fi;
```

Comment:
Can we make these two links an anchor? (with _target=blank)

Reply:
I initially tried sending the payload in markdown through GitHub's API and it didn't render correctly because endlines were not parsed (I tried ``\n\`` and ``\r\n``). So the only option was to build simple HTML with <br> to create the newlines.

As per the anchor links, they do not render properly either, you can try editing a comment by the bot and trying to add them there before editing the script.

Reply:
Comment edited: https://github.com/EyeSeeTea/metadata-synchronization/pull/452#issuecomment-600042369

Regarding the line break, untested: https://stackoverflow.com/a/6262430/188031

Reply:
Okay cool, then I will do the change now, not sure why it failed on my test then